<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>1071</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0,0,0">
      <item>
       <widget class="QPushButton" name="pushButton_details">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Display Details</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_details">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_images">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Display Image Samples</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>98</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_selectFolder">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select Folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Path">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_model">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select a model:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">resnet50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">mobilenet_v3_large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">vgg16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">vgg19</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adam</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SDG</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select Optimizer</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
